### PR TITLE
Add gfp_rel_series sqrt functions and deal with characteristic 2

### PR DESCRIPTION
**Pull request by wbhart**
Requires flint-2.9.

### CODE REVIEW

**Comment by tthsqe12** [comment]
this one has a problem

File: src/flint/nmod_rel_series.jl [hunk 1]
```diff
@@ -945,32 +945,99 @@ end
...
+      end
+   end
+   for i = 0:prec - aval2 - 1
+      asqrt = setcoeff!(asqrt, i, polcoeff(a, 2*i))
+   end
+   return true, asqrt
+end
+
+function sqrt_classical(a::gfp_rel_series; check::Bool=true)
+   S = parent(a)
```
Comment (tthsqe12) [inline src/flint/nmod_rel_series.jl:987]:
```suggestion
   S = parent(a)
   if characteristic(S) == 2
      return sqrt_classical_char2(a; check=check)
   end

```

**Review by tthsqe12 — CHANGES_REQUESTED** [review]

File: src/flint/nmod_rel_series.jl (hunk 1 above)
Comment (thofma) [reply src/flint/nmod_rel_series.jl:987]: I can't commit those changes.

Comment (tthsqe12) [reply src/flint/nmod_rel_series.jl:987]: Neither can I

**Pull request merged**